manageiq-operator/README.md

```diff
@@ -1,28 +1,21 @@
-ManageIQ Operator
-------
-This operator manages the lifecycle of ManageIQ application on a OCP4 cluster.
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+**Table of Contents**  *generated with [DocToc](https://github.com/thlorenz/doctoc)*
```

Comment:
This doesn't really seem necessary for a doc of this size.

Reply:
As I was using `doctoc` and it was generated automatically, how about keep this? As we may need to update the toc later.

If we remove this line, and then when we run `doctoc`, this line will be added back automatically.

Reply:
I think @carbonin is suggesting to just not run doctoc.  Otherwise, this becomes a dependency for developers writing the README, and for a simple README it seems unnecessary.

Reply:
Ah, I see, let me remove this